src/Go.Windows/Logger.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace Go.Windows
```

Comment:
Can we move this file into a `Hosting` folder/namespace? Various extension projects use this folder as a home for anything that needs to be implemented light up the shared MEF part.